src/Pages/homePage.js

```diff
@@ -0,0 +1,82 @@
+import React from 'react';
+import { v4 as uuidv4 } from 'uuid';
+import { Box, Stack, useTheme } from '@mui/material';
+import { useSelector, useDispatch } from 'react-redux';
+import SearchInput from '../components/SearchInput';
+import SelectContinent from '../components/SelectContinent';
+import CountryCards from '../components/CountryCards';
+import { getCountriesFromServer } from '../redux/Countries/countries';
+// import { RootState } from '../redux/configureStore';
+
+const Homepage = () => {
+  const theme = useTheme();
+  const countries = useSelector((state) => state.countries);
+  const selectedContinent = useSelector((state) => state.selectedContinent);
+  // console.log(selectedContinent);
+  const dispatch = useDispatch();
+  React.useEffect(() => {
+    dispatch(getCountriesFromServer(selectedContinent));
+  }, [selectedContinent]);
+  return (
+    <Box
+      component="main"
+      sx={{
+        mt: '2rem',
+        p: '2rem 4rem',
+        [theme.breakpoints.down('sm')]: {
+          p: '0rem',
+        },
+      }}
+    >
+      <Stack sx={{
+        flexDirection: 'row',
+        justifyContent: 'space-between',
+        [theme.breakpoints.down('md')]: {
+          flexDirection: 'column',
+          alignItems: 'center',
+          gap: '30px',
+        },
+      }}
+      >
+        <SelectContinent continent={selectedContinent} />
+        <SearchInput />
+      </Stack>
+      <Stack sx={{
+        mt: '2rem',
+        flexWrap: 'wrap',
+        flexDirection: 'row',
+        justifyContent: 'center',
+        // gap: '3rem',
+        // [theme.breakpoints.down('sm')]: {
+        //   flexDirection: 'column',
+        //   alignItems: 'center',
+        //   gap: '2rem',
+        // },
+      }}
+      >
+        {
+          countries
+            ? countries.map((country) => (
+            // eslint-disable-next-line react/jsx-key
+              <CountryCards
+                name={country.name}
+                flag={country.flag}
+                population={country.population}
+                capital={country.capital}
+                continents={country.continents}
+                languages={country.languages}
+                nativeName={country.nativeName}
+                currencies={country.currencies}
+                subregion={country.subregion}
+                topLevelDomain={country.topLevelDomain}
+                key={uuidv4()}
+              />
```

Comment:
- [ ] Kindly stick to the original design as much as possible. Please make sure that each row showcases two items instead of just one item on mobile widths (width: 375px).

| Your design | Original design |
| --- | --- |
| ![image](https://user-images.githubusercontent.com/62290545/189145696-51d3512b-7072-4e0c-a306-b1eff109fadb.png) | ![image](https://user-images.githubusercontent.com/62290545/189145813-86c79044-7e01-4b42-b6d6-5d89b8e729cd.png) |

Reply:
@ShadyShawkat  Kindly check out eployed version it is showcasing two items in each row
![image](https://user-images.githubusercontent.com/104348397/189157738-815f54a5-eee2-4813-96ca-890dce4194bf.png)
Please check it out and update asap!

Reply:
@ishpaul777, This is what I'm seeing in the live link 
![image](https://user-images.githubusercontent.com/62290545/189159496-12f1aa82-3a4f-4a06-80c0-fe0984321b75.png)

Reply:
Would you mind joining a zoom call

Reply:
It will be quick @ShadyShawkat

Reply:
I can't join a meeting as of right now. We can talk about this more on slack if you want.

Reply:
You can see the live video its showing 2 items on 375 width iphone screen

Reply:
This requested change has been deprecated. Please disregard it.